build/cmake/init.cmake

```diff
@@ -50,6 +50,15 @@ if(wxBUILD_COMPATIBILITY VERSION_LESS 3.1)
     set(WXWIN_COMPATIBILITY_3_0 ON)
 endif()
 
+if(wxUSE_NO_RTTI)
+    if(MSVC)
+        set(RTTI_COMPILER_FLAGS "/GR-")
+    elseif(CMAKE_COMPILER_IS_GNUCC)
```

Comment:
It is [suggested](https://cmake.org/cmake/help/latest/variable/CMAKE_COMPILER_IS_GNUCC.html) to use `"${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU"`. 
Also relevant for your other PRs.

Reply:
OK, thank you. I will fix it.

Reply:
Done.